### PR TITLE
test: update versions for upgrade tests

### PR DESCRIPTION
**Pull request by smira**
Use the latest releases in each branch.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

**Review by frezbo — APPROVED** [review]

File: pkg/cluster/check/kubernetes.go [hunk 1]
```diff
@@ -411,6 +412,11 @@ func K8sControlPlaneStaticPods(ctx context.Context, cl ClusterInfo) error {
 			resource.NewMetadata(k8s.NamespaceName, k8s.StaticPodStatusType, "", resource.VersionUndefined),
 		)
 		if err != nil {
+			if client.StatusCode(err) == codes.Unimplemented {
```
Comment (smira) [inline pkg/cluster/check/kubernetes.go:415]: skips the new check introduced recently for Talos < 1.2 (in the upgrade tests we run health check for v1.1)

**Pull request merged**